### DISCUSSION TITLE
Installation Directions Do Not Work For Windows 10 Enterprise

### DISCUSSION THREAD

**Discussion by SteveRussellVA** [coreybutler/nvm-windows#1099] [General]
I downloaded a fresh copy of the installer and that worked.

Please feel free to remove this thread.

Thanks much for the response.

**Comment by coreybutler** [comment]
That sounds very odd. I know many other users have used that combination, so I'm not sure what would be different. My gut reaction is a permission issue... but if you're installing as an admin, that doesn't seem right.

Are you using the official installer? Nothing sets an empty ENV variable in the official installer.

**Reply by SteveRussellVA** [reply]
I downloaded a fresh copy of the installer.  It worked this time.  My apologies.
